tests/integration/cmd_link_unlink_test.go

```diff
@@ -24,10 +24,10 @@ var _ = Describe("odoLinkE2e", func() {
 	// This is before every spec (It)
 	var _ = BeforeEach(func() {
 		SetDefaultEventuallyTimeout(10 * time.Minute)
-		oc = helper.NewOcRunner("oc")
-		project = helper.CreateRandProject()
 		context = helper.CreateNewContext()
 		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+		oc = helper.NewOcRunner("oc")
+		project = helper.CreateRandProject()
```

Comment:
Now that it has shown up thrice in the PR, I'm sure there's some significance to changing the order of defining these variables. Out of curiosity, what is it?

Reply:
https://github.com/openshift/odo/pull/1912/files#r308234314